src/Symfony/Component/HttpKernel/EventListener/ErrorListener.php

```diff
@@ -123,7 +123,7 @@ protected function duplicateRequest(\Exception $exception, Request $request)
     {
         $attributes = [
             '_controller' => $this->controller,
-            'exception' => FlattenException::createFromThrowable($exception),
+            'exception' => $exception,
```

Comment:
Now apps with a custom exception controller must update its typehint to `\Exception` and probably do `FlattenException::createFromThrowable($exception)` on their own. It might require an UPGRADE note.

Reply:
Fixed by making ErrorListener turn Throwable into FlattenException when the type declaration of the controller requires so.
See `ErrorListener::onControllerArguments()`

Reply:
It works for me! thanks.